catalyst_voices/packages/catalyst_voices_models/lib/src/errors/crypto_exception.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:equatable/equatable.dart';
+
+sealed class CryptoException extends Equatable implements Exception {
+  const CryptoException();
+
+  @override
+  List<Object?> get props => [];
+}
+
+/// Usually thrown when trying to decrypt with invalid key
+final class CryptoAuthenticationException extends CryptoException {
+  const CryptoAuthenticationException();
+
+  @override
+  String toString() => 'CryptoAuthenticationException';
+}
+
+/// Thrown when data trying to decrypt was tempted with
+final class CryptoDataMalformed extends CryptoException {
+  final String? message;
+
+  const CryptoDataMalformed([this.message]);
+
+  @override
+  String toString() {
+    if (message != null) return 'CryptoDataMalformed: $message';
+    return 'CryptoDataMalformed';
+  }
+}
+
+final class CryptoVersionUnsupported extends CryptoException {
+  final String? message;
```

Comment:
Shouldn't these messages by part of the equatable props?

Reply:
Yes, it should. Forgot about it.